tests/functional_tests/exporter_test.yaml

```diff
@@ -26,7 +26,7 @@
 
```

Comment:
CLA not approvable as commit has been performed using a different email than expected by CLA bot. Close this PR and raise a new one please.